src/Simulator/IncrementalRedstoneSimulator/ForEachSourceCallback.cpp

```diff
@@ -49,6 +49,36 @@ void ForEachSourceCallback::operator()(Vector3i Location)
 		return;
 	}
 
+	// Quasiconnectivity handling
+	// If the block is viable and the source offset is elite:
```

Comment:
Can you de-thesaurify this comment for me?

Reply:
elite, like e1337 :p

but apparently elitism isn't popular here so I'll stick this in special function to handle QC